### ISSUE TITLE
There are spaces around operators most of the time, but not here.

### ISSUE THREAD

**Issue by jweinkam** [rrrene/credo#797]
### Precheck

* Proposals for new features should be submitted via: https://github.com/rrrene/credo-proposals
* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): 1.4.0
* Erlang/Elixir version (`elixir -v`): Erlang/OTP 22 [erts-10.6.4] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe]

Elixir 1.8.2 (compiled with Erlang/OTP 22)
* Operating system: macOS Catalina Version 10.15.5 (19F101)

### What were you trying to do?

    compare_fn = &max/2
    b = 5
    c = 10
    d = 4

    a = b + c + compare_fn.(-d, 0)

### Expected outcome

No Consistency problems

### Actual outcome

[C] ↗ There are spaces around operators most of the time, but not here.

It appears to be complaining about the unary '-'.

**Comment by crqra** [comment]
I'm also facing this issue. Credo complies on statements like:

```
parse_header(rest, fields ++ [{:comment, comment}])
```

As far as I could test, it happens because I have lot's of matches like this: `min_ver::size(16)-unsigned-integer-little`.

If I put a space between the `-` in those statements like `min_ver::size(16) - unsigned - integer - little`, even if it's only in one of them, the consistency warning goes away.

**Comment by crqra** [comment]
Interestingly, `mix format` will keep `min_ver::size(16)-unsigned-integer-little` without spacing. However on IntelliJ if I format the code it separates them: `min_ver :: size(16) - unsigned - integer - little`.

What's the best approach?

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by jweinkam** [comment]
I have tested the version from master with this function 
```
  def test() do
    compare_fn = &max/2
    b = 5
    c = 10
    d = 4

    a = b + c + compare_fn.(-d, 0)
  end
```
and it is still complaining about the negative sign on the line `a = b + c + compare_fn.(-d, 0)`

If I change the code to 
```
  def test() do
    compare_fn = &max/2
    b = 5
    c = 10
    d = -4

    a = b + c + compare_fn.(d, 0)
  end
```
then is stops complaining.

**Comment by rrrene** [comment]
@jweinkam Thx for clarifying this! :+1: Could you check if it is fixed in `1.5.0-rc.5`?

**Comment by jweinkam** [comment]
I retested today with 1.5-rc.5 and it is now accepting the unary minus.  Thank you.
